### PR TITLE
delete manually-maintained contributors list & add approval details in governance doc

### PR DESCRIPTION
**Pull request by milasuperstar**
### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] No code
- [ ] Manual test (add detailed scripts or steps below)

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by zhouqiang-cl — DISMISSED** [review]
LGTM

**Comment by milasuperstar** [comment]
@dcalvin PTAL

File: GOVERNANCE.md [hunk 1]
```diff
@@ -67,20 +67,25 @@ If a maintainer is no longer interested or cannot perform the maintainer duties
...
 
 # Decision Making Process
 
-Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus model](https://communitymgt.fandom.com/wiki/Lazy_consensus). Community-level decisions must be brought to broader awareness via communications such as community meetings and slack channels.
+Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus model](https://communitymgt.fandom.com/wiki/Lazy_consensus). Community-level decisions such as:
+
+- RFC approvals (see below)
+- Modifying GOVERNANCE.md
+
+must be brought to broader awareness via communications such as community meetings and slack channels and supermajority (2/3) approval from Maintainters is required.
```
Comment (dcalvin) [inline GOVERNANCE.md:83]:
```suggestion
Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. Any of the day-to-day project maintenance can be done by a [lazy consensus model(https://communitymgt.fandom.com/wiki/Lazy_consensus).
Community or project level decisions such as RFC submission, creating a new project, maintainer promotion,  and major updates on GOVERNANCE must be brought to broader awareness of the community via community meetings, GitHub discussions,  and slack channels. A supermajority (2/3) approval from Maintainers is required for such approvals.
```

Comment (milasuperstar) [reply GOVERNANCE.md:83]: @yeya24 PTAL:)

Comment (yeya24) [reply GOVERNANCE.md:83]: Thanks for the work. This looks perfect!

**Review by yeya24 — COMMENTED** [review]
LGTM

**Review by dcalvin — DISMISSED** [review]
LGTM

**Comment by milasuperstar** [comment]
@dcalvin @cwen0 @zhouqiang-cl 
Final version, PTAL:)

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by STRRL — APPROVED** [review]

**Review by yeya24 — APPROVED** [review]
LGTM

**Pull request merged**